android/KMEA/app/src/main/assets/android-host.js

```diff
@@ -128,7 +133,8 @@ function notifyHost(event, params) {
 // Update the KeymanWeb longpress delay
 // delay is in milliseconds
 function setLongpressDelay(delay) {
-  if (keyman.osk) {
+  longpressDelay = delay;
+  if (keyman && keyman.osk) {
     keyman.osk.gestureParams.longpress.waitLength = delay;
     console_debug('setLongpressDelay('+delay+')');
   } else {
```

Comment:
This `else` clause is not really needed because we have the pre-init and post-init pathways defined